tools/genomad/genomad_end_to_end.xml

```diff
@@ -59,6 +59,7 @@
             '$DATABASE.fields.path'
     ]]></command>
     <inputs>
+        <param name="license" type="boolean" checked="false" label="Agree with license" help="geNomad is free to use for internal use, research &amp; development, non-commercial use, purposes only. License is linked in the help below."/>
```

Comment:
You are missing a validator here. And maybe formulate it more like a question? https://github.com/galaxyproject/tools-iuc/blob/main/tools/meme/fimo.xml